api/package.json

```diff
@@ -47,17 +46,19 @@
     "jest": "^24.8.0",
     "prettier": "^1.18.2",
     "ts-jest": "^24.0.2",
+    "tsconfig-paths": "^3.8.0",
     "tslint-config-prettier": "^1.18.0",
     "tslint-microsoft-contrib": "^6.2.0",
-    "typescript": "^3.5.2"
+    "typescript": "3.3.4000"
```

Comment:
Why this version?

Reply:
https://github.com/Microsoft/TypeScript/issues/29112#issuecomment-486107941

i tried later versions with no luck